plugins/font/plugin.js

```diff
@@ -46,7 +46,8 @@
 						left: function( element ) {
 							return !!element.attributes.size ||
 								!!element.attributes.align ||
-								!!element.attributes.face;
+								!!element.attributes.face ||
+								!!element.styles[ 'font-size' ];
```

Comment:
Looks like I forgot to push this topic further. Created a separate ticket for this http://dev.ckeditor.com/ticket/16977.